tests/migration/recovery.go

```diff
@@ -147,8 +148,10 @@ var _ = Describe("[sig-compute]Migration recovery", decorators.SigCompute, decor
 		Expect(err).NotTo(HaveOccurred())
 		Expect(pvc.Labels).To(HaveKeyWithValue("persistent-state-for", vmi.Name))
 	},
-		Entry("failure", false),
-		Entry("success", true),
+		Entry("failure", decorators.NoFlakeCheck, false, false),
+		Entry("success", decorators.NoFlakeCheck, true, false),
+		Entry("failure [Serial]", decorators.FlakeCheck, Serial, false, true),
+		Entry("success [Serial]", decorators.FlakeCheck, Serial, true, true),
```

Comment:
nit: the `[Serial]` is not needed anymore.

Reply:
This is actually to ensure those tests don't have the same name as the previous 2, since prow indexes on test name.